deployments/sherlock-frontend/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: sherlock
+
+resources:
+  - github.com/lsst-sqre/sherlock-frontend.git//manifests/base?ref=0.0.4
```

Comment:
Just a heads-up the 0.0.4 tag doesn't exist yet.

Reply:
Yep going to make that with this other PR.  Just wanting to make sure we want the ingress over here rather than over there.  If so, then I'll do the merge (and probably have it broken)